src/common.rs

```diff
@@ -99,6 +108,24 @@ pub fn get_restful_parameters(urlstring: &str) -> HashMap<&str, &str> {
     parameters
 }
 
+/*
+ * Input: path directory
+ *
+ * If previledge requirement is met, change the owner of the path to root
+ * This function is unsafely using libc
+ */
+#[allow(dead_code)]
+pub fn chownroot(path: String) {
+    unsafe {
+        let euid = libc::geteuid();
+        if euid == 0 {
+            libc::chown(path.as_bytes().as_ptr() as *const i8, 0, 0);
```

Comment:
Handle return code on this.

Reply:
A status code is returned to caller with execution result log.